CryptoExchange.Net/CryptoExchange.Net.csproj

```diff
@@ -19,15 +19,16 @@
     <PackageIcon>icon.png</PackageIcon>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageReleaseNotes>https://github.com/JKorf/CryptoExchange.Net?tab=readme-ov-file#release-notes</PackageReleaseNotes>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Nullable>enable</Nullable>
     <LangVersion>12.0</LangVersion>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
   <ItemGroup>
     <None Include="Icon\icon.png" Pack="true" PackagePath="\" />
     <None Include="..\README.md" Pack="true" PackagePath="\" />
   </ItemGroup>
-  <PropertyGroup Label="AOT" Condition=" '$(TargetFramework)' == 'NET8_0' Or '$(TargetFramework)' == 'NET9_0' ">
```

Comment:
It's doubtful this condition will ever be evaluated as `true`. `$(TargetFramework)` contains the exact TFM string that MSBuild gets from your [<TargetFrameworks>](https://learn.microsoft.com/en-us/dotnet/standard/frameworks) list. The strings `NET8_0`, `NET9_0` are the C# pre‑processor symbols that the compiler defines for `#if NET8_0` blocks inside code. They never flow into MSBuild properties.